tutorials/intermediate/constrainedlayout_guide.py

```diff
@@ -105,7 +105,7 @@ def example_plot(ax, fontsize=12, nodec=False):
 fig.colorbar(im, ax=ax, shrink=0.6)
 
 ############################################################################
-# If you specify multiple axes to the ``ax`` argument of ``colorbar``,
+# If you specify a list of axes to the ``ax`` argument of ``colorbar``,
```

Comment:
If I understood correctly, even a tuple or an ndarray (or actually any kind of iterable...) will trigger that behavior, will it not? Should this be briefly mentioned or would it make the API even harder to understand :/?